src/app/address/steps.js

```diff
@@ -14,7 +14,15 @@ module.exports = {
   "/search": {
     controller: search,
     fields: ["address-search"],
-    next: "results",
+    next: [
+      {
+        field: "isSuccessful",
+        op: (fieldValue, req, res, con) => fieldValue === con.value,
```

Comment:
I don't think this needs to be this complicated - you should be able to a field & value combination and the op defaults to `===` - https://github.com/HMPO/hmpo-form-wizard#next-steps